### ISSUE TITLE
[Feature Request] Add syntax completion to nvim

### ISSUE THREAD

**Issue by eyalk11** [dccsillag/magma-nvim#101]
One would like to inspect his variables to see their attributes. So it is beneficial to have it as additional lsp source  or provide a custom completion function that is aligned with jupyter namespace. 
In terms of what I want to happen is writing `"my_variable.<tab>"` would show its methods etc . When relying on static sources such as mypy , some of it might be wrong or simply unresolved.

**Comment by WhiteBlackGoose** [comment]
Hi. I use [this](https://github.com/lkhphuc/jupyter-kernel.nvim) for this purpose, but probably might make sense to merge it in magma too

**Comment by eyalk11** [comment]
Can you work with both at the same time? I guess you can just add it as a dependency of your project and call JupyerAttach

**Comment by WhiteBlackGoose** [comment]
You can work with both at the same time yes. I start jupyter process separately and then connect to it from both

**Comment by dccsillag** [comment]
Perhaps it would be worth it to add a helper to connect both in a single command. (Even if just as a suggested snippet in the README.)

**Comment by WhiteBlackGoose** [comment]
Yeah. Last time I tried, for some reason it didn't work (probably bc simultaneous connections?), worth probably to try again

**Comment by dccsillag** [comment]
Yep, that sounds likely. IIRC, Magma triggers a user autocommand when the kernel connects, so that may be of help.
(Also, maybe having a loop to try connecting a couple of times [~3ish], waiting a bit in between attempts, before giving up?)
